tests/queries/0_stateless/03393_join_bug_77848.sql

```diff
@@ -0,0 +1,16 @@
+-- Tags: no-parallel-replicas
```

Comment:
Please could you try instead?
```
SET parallel_replicas_local_plan=1;
```

Reply:
https://github.com/ClickHouse/ClickHouse/pull/78274